_tools/merge_docs.sh

```
@@ -0,0 +1,20 @@
+# 스크립트 위치를 기반으로 프로젝트 루트로 이동
+cd "$(dirname "$0")/.." || exit 1
+
+# 이후는 기존대로 진행
+OUTPUT_DIR="_tools"
+OUTPUT_FILE="$OUTPUT_DIR/project_standard.md"
+
+mkdir -p "$OUTPUT_DIR"
+: > "$OUTPUT_FILE"
+
+for FILE in $(find docs -type f -name "*.md" | sort); do
+  echo "# 📄 $FILE" >> "$OUTPUT_FILE"
```

Shellcheck (0.10.0)

[style] Consider using { cmd1; cmd2; } >> file instead of individual redirects.

(SC2129)